### PR TITLE
Make tab width configurable for `LineLengthCheck`

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
This is the last property on a check that couldn't be changed (besides the one in #98).

Also changed `CommentedCodeCheck` to have a uniform styling.

### CODE REVIEW

File: magik-checks/src/main/java/nl/ramsolutions/sw/magik/checks/checks/LineLengthCheck.java [hunk 1]
```diff
@@ -45,8 +54,8 @@ protected void walkPreMagik(final AstNode node) {
       for (int i = 0; i < line.length(); ++i) {
         final char chr = line.charAt(i);
         if (chr == '\t') {
-          final int mod = columnNo % TAB_WIDTH;
-          columnNo += TAB_WIDTH - mod;
+          final int mod = columnNo % tabWidth;
```
Comment (StevenLooman) [inline magik-checks/src/main/java/nl/ramsolutions/sw/magik/checks/checks/LineLengthCheck.java]: Please use `this.tabWidth` for instance variables. This makes it easier to determine where the variable comes from.

Comment (sebastiaanspeck) [inline magik-checks/src/main/java/nl/ramsolutions/sw/magik/checks/checks/LineLengthCheck.java]:
```suggestion
          final int mod = columnNo % this.tabWidth;
```

File: magik-checks/src/main/java/nl/ramsolutions/sw/magik/checks/checks/LineLengthCheck.java [hunk 2]
```diff
@@ -45,8 +54,8 @@ protected void walkPreMagik(final AstNode node) {
...
+          columnNo += tabWidth - mod;
```
Comment (sebastiaanspeck) [inline magik-checks/src/main/java/nl/ramsolutions/sw/magik/checks/checks/LineLengthCheck.java]:
```suggestion
          columnNo += this.tabWidth - mod;
```

**Comment by StevenLooman** [comment]
Thanks!

**Pull request merged**